samples/world-clock/presenters/src/androidMain/kotlin/app/cash/zipline/samples/worldclock/android.kt

```diff
@@ -54,7 +54,6 @@ class WorldClockAndroid(
   }
 
   fun close() {
-    scope.cancel()
     ziplineExecutorService.shutdown()
```

Comment:
We should be able to tie this to the scope so no `close()` is needed. Will try in a follow-up.